docs/PAYLOAD.md

```diff
@@ -16,7 +16,7 @@
   - [Priority in Notifications](#priority-in-notifications)
   - [Picture Messages](#picture-messages)
   - [Background Notifications](#background-notifications)
-    - [Use of content_available: true](#use-of-content-available-true)
+    - [Use of content-available: true](#use-of-content-available-true)
```

Comment:
The wrong part of this bad link is the ID, not the name. This should instead be fixed to:
`- [Use of content_available: true](#use-of-content_available-true)`

Reply:
ok, fixed